extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1090-lms-assignment-submissions.sql

```diff
@@ -41,6 +41,40 @@ BEGIN
 		ON EDFISTUDENT.Id = lmsUser.EdFiStudentId
     WHERE lmsSubmission.SourceSystem = @SourceSystem
 
+	INSERT INTO #ALL_SUBMISSIONS
+	SELECT
+		edfisectionassociation.SectionIdentifier
+			+ '#' + lmsxassignment.AssignmentIdentifier
+			+ '#' + lmsstudent.SourceSystemIdentifier
+			as AssignmentSubmissionIdentifier,
+
+		edfisectionassociation.StudentUSI,
+		lmsxassignment.AssignmentIdentifier,
+		submsisionstatusdescriptor.DescriptorId,
+		NULL as SubmissionDateTime,
+		0 as EarnedPoints,
+		'F' as Grade,
```

Comment:
Although `0` and `F` make sense, I'm not sure that we should assign a grade here. This might be one of the differences between the LMS assignments and the SIS gradebook entries. We should look to see what we get from Canvas and Google Classroom and try to be consistent with that.